src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,52 @@
+package duke.command;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+import duke.exception.InvalidInputException;
+import duke.task.Task;
+
+/**
+ * Represents a DeleteCommand and handles methods related to commands
+ * about deleting a task.
+ */
+public class DeleteCommand extends Command {
```

Comment:
Perhaps the class name could change to CommandDeleter or other nouns? Reference: Use nouns for classes/variables and verbs for methods/functions.

Reply:
Ohh I think I misinterpret this as a verb at first, but from the rest of the class names,  I could tell this is a noun as well

Reply:
Okay thanks!